src/fsharp/ast.fs

```diff
@@ -1365,18 +1366,18 @@ and SynModuleDecls = SynModuleDecl list
 and 
     [<NoEquality; NoComparison>]
     SynExceptionSig = 
-    | SynExceptionSig of SynExceptionDefnRepr * SynMemberSigs * range
+    | SynExceptionSig of SynExceptionDefnRepr * SynMemberSigs * range:range
 
 and
     [<NoEquality; NoComparison; RequireQualifiedAccess>]
     SynModuleSigDecl =
-    | ModuleAbbrev      of Ident * LongIdent * range
-    | NestedModule      of SynComponentInfo * isRec : bool * SynModuleSigDecls * range
-    | Val               of SynValSig * range
-    | Types             of SynTypeDefnSig list * range
-    | Exception         of SynExceptionSig * range
-    | Open              of LongIdent * range
-    | HashDirective     of ParsedHashDirective * range
+    | ModuleAbbrev      of ident:Ident * longId:LongIdent * range:range
+    | NestedModule      of SynComponentInfo * isRecursive:bool * SynModuleSigDecls * range:range
+    | Val               of SynValSig * range:range
```

Comment:
@cloudRoutine If you can do the integrations visualfsharp --> fsharp --> FCS then that would help, though it's not easy to do those.

You should first integrate all changes _besides_ these ones from VF --> F -> FCS

Then make the precise aligning change to VF which gets the diff smaller

Then integrate again VF --> F --> FCS

Note the VF doesn't contain the commit history from F or FCS (and indeed can't contain those for various legal and technical reasons).  So we can't integrate FCS --> VF or F --> VF.  But F and FCS do contain the commit history from VF.